python/guide/packages/gluon/data.md

```diff
@@ -1,325 +1,120 @@
 # Data
 
-The `data` module provides APIs to 1) load and parse datasets, 2)
-transform data examples, and 3) sample mini-batches for the training program.
-This tutorial will go through these three functionalities. Let's first import
-the modules needed, the key module is `mxnet.gluon.data`, which is imported as
-`gdata` to avoid the too commonly used name `data`.
-
-```{.python .input  n=1}
-import numpy as np
-from matplotlib import pyplot as plt
-import tarfile
-import time
-
-from mxnet import nd, image, io
-from mxnet.gluon import utils, data as gdata
-```
+One of the most critical steps for model training and inference is loading the data: without data you can’t do deep learning! In this tutorial we use the `data` module to:
 
-## Load and Parse Datasets
+1) define a [`Dataset`](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.data.Dataset.html)
```

Comment:
I think upper-case looks better in the list:
1) Define ...
2) Use ...
3) Use ...

Reply:
Done.